src/server.ts

```diff
@@ -13,7 +13,7 @@ const useSSL = process.env.SSL_ENABLED === "true";
 const port = process.env.WEB_EVENTS_PORT ? process.env.WEB_EVENTS_PORT : useSSL ? 8443 : 80;
 
 const corsOptions: CorsOptions = {
-  origin: process.env.WEBAPP_URL || "http://localhost:3334",
+  origin: process.env.WEBAPP_URL || `http://localhost`,
```

Comment:
make sure we can remove this PORT and that all environments have a correct value for `WEBAPP_URL`